### ISSUE TITLE
Submission 02 Neural Network

### ISSUE THREAD

**Issue by wildanaziz** [wildanaziz/TL-Vision#15]
# -- Mitos Filkom Memprediksi Outfit Anak Filkom --

Mitos Filkom merupakan seorang mahasiswa semester 4 di prodi yang katanya paling teknik di Filkom. Di semester 4 ini Mitos Filkom sedang mengambil mata kuliah Sistem Cerdas. Dimana pada mata kuliah tersebut Mitos Filkom diminta untuk membuat suatu projek akhir terkait memprediksi outfit yang sering dipakai oleh anak Filkom menggunakan algoritma Neural Network. Dimana Algoritma Neural Network yang pernah diajarkan oleh dosen di mata kuliah tersebut meliputi MLP (Multi Layer Perceptron) dan CNN (Convolutional Neural Network). Vision Amarine peers diminta untuk membantu Mitos Filkom menyelesaikan side projectnya terkait memprediksi outfit yang sering dipakai anak filkom berdasarkan dataset mnist_fashion yang terlampir pada keterangan. Tugas vision amarine peers detailnya seperti berikut:

1. Membuat algoritma MLP berdasarkan pembelajaran pada week-3.
2. Membuat algortima CNN menggunakan tensorflow.
3. Melatih serta memprediksi gambar berdasarkan algoritma MLP yang telah dibuat.
4. Melatih serta memprediksi gambar berdasarkan algoritma CNN yang telah dibuat.
5. Bandingkan hasil algoritma antara MLP dan CNN berdasarkan hasil evaluasi metric akurasi.

Keterangan:

1. dataset_fashion_mnist: https://www.tensorflow.org/datasets/catalog/fashion_mnist
dataset_fashion_mnist: https://www.kaggle.com/datasets/zalando-research/fashionmnist
2. Untuk poin 1-5 vision amarine peers harus mencantumkan laporan proses pengerjaan dalam bentuk README.md disetiap repository submission yang dibuat
3. Untuk poin 1 vision amarine peers bisa melihat video tutorial terkait MLP di week-3
4. Untuk poin 2 vision amarine peers bisa melihat video tutorial terkait CNN di week-4
5. Pengerjaan bersifat berkelompok, vision amarine peers dapat membagi tugas antar individu agar dapat menyelesaikan tugas berikut dan diharuskan untuk menuliskan kontribusi pada poin berapa yang dikerjakan pada README.md.contoh struktur directory submission (boleh ditiru)

```markdown 
week-4/ (ini direktori TL-Vision)
├── submission/ (ini direktori TL-Vision)
│ └── siGondrong/ (Ngumpulin sesuai namamu)
│ ├── projek_main/ (Ini yang ngerjain bareng)
│ ├── README.md
│ └── main.py
│ └── projek_poin_siGondrong (Kamu ngerjain bagian mana aja)
│ ├── README.md
│ └── main.py
```
